cost-analyzer/values.yaml

```diff
@@ -2914,6 +2983,9 @@ federatedETL:
   ## Prometheus)
   useMultiClusterDB: false
 
+  ## Define if this is the primary cluster
+  primaryCluster: false
```

Comment:
I'd recommend removing this, as this is only a value used in legacy FedETL installations. As of 2.x there is no concept of a primary FedETL cluster. Instead, the Aggregator should be run on the cluster which was previously designated the primary. Overall, these changes look great though!

Reply:
Ah thank you, that shouldn't have been there. I think when I resolved conflicts it got included somehow. Fixed.